mantidproject/mantid
# Restore axis tick format on project save and load

**Description of work.**

Created a dictionary of supported tick formats and applied the correct format to plot axes ticks when the project is loaded.

If an axis tick format is not available in the above dictionary, 'ScalarFormatter' is set as the default format.



**To test:**


-> Load a file
-> Plot one or more of the spectra from it
-> Change an axis tick format by clicking on the axis.  As an example, change an axis tick format to 'scientific format'.
-> Save the project and re-open it

The tick format should be the same (scientific) format as set previously.

Fixes #31259.







---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
